react/src/components/SourceCodeViewer.tsx

```diff
@@ -0,0 +1,91 @@
+import { useWebComponentInfo } from './DefaultProviders';
+import Flex from './Flex';
+import { CheckOutlined, CopyOutlined } from '@ant-design/icons';
+import { Button } from 'antd';
+import { Highlight, themes } from 'prism-react-renderer';
+import { useRef, useState } from 'react';
+// @ts-ignore
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+
+interface SourceCodeViewerProps {
+  // In case of using this component in a web component, the props should be parsed from the web component.
+  codeBlock?: string;
+  language?: string;
+  fontSize?: string;
+}
+
+const SourceCodeViewer: React.FC<SourceCodeViewerProps> = ({
```

Comment:
`SourceCodeViewer` itself should NOT be implemented assuming it will be used within a web component. Implement SourceCodeViewer to receive necessary input through props instead of `useWebComponentInfo`. For web component, we can make it High order component like [this](https://github.com/lablup/backend.ai-webui/blob/065f3e2d88a0ccb4c86b12d62e9d4adf976fab8a/react/src/index.tsx#L172-L184). @ironAiken2